core-feature-pack/common/src/main/resources/modules/system/layers/base/org/apache/httpcomponents/core/main/module.xml

```diff
@@ -34,7 +34,9 @@
 
 
     <dependencies>
-        <module name="javax.api"/>
+        <!--WFCORE-5286 Remove deprecated <module name="javax.api"/> -->
```

Comment:
I think these lines should just be removed if we are removing them, we don't send to leave our files with a build up of commented out items unless we really need to keep the removal visible in the future e.g. commenting out a redundant error code to stop it being used again in the future.

Reply:
@darranl thanks for your review. I have removed the commented line in all cases. Please review again.